src/dotnet/Chat.UI.Blazor/Pages/ChatPage.razor

```diff
@@ -48,15 +51,17 @@
 }
 
 @code {
+    private static string? _deviceToken;
```

Comment:
This will for sure fuck up everything in Blazor Server. You need to move this stuff to a scoped service.

Reply:
Thanks!